templates/beego/files/controllers/default.go

```diff
@@ -0,0 +1,15 @@
+package controllers
+
+import (
+	"github.com/astaxie/beego"
+)
+
+type MainController struct {
+	beego.Controller
+}
+
+func (c *MainController) Get() {
+	c.Data["Website"] = "beego.me"
+	c.Data["Email"] = "astaxie@gmail.com"
```

Comment:
maybe: `demo@mycompany.com`

Reply:
I don't know about this one. It's the starter template for this whole framework just put on Platform, not sure how much we should change the front page/links. That is the development team's email.